portals/publisher/src/main/webapp/site/public/locales/en.json

```diff
@@ -9055,6 +9501,12 @@
       "value": "List of APIs"
     }
   ],
+  "Select.Gateways": [
+    {
+      "type": 0,
+      "value": "Select Gateways to Deploy"
```

Comment:
```suggestion
      "value": "Select gateways to deploy"
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)